src/encoded/static/components/genome_browser.js

```diff
@@ -0,0 +1,432 @@
+import React from 'react';
+
+
+const maxFilesBrowsed = 5; // Maximum number of files to browse
+const domainName = 'https://www.encodeproject.org';
+const dummyFiles = [
+    {
+        file_format: 'bigWig',
+        output_type: 'minus strand signal of all reads',
+        accession: 'ENCFF425LKJ',
+        href: '/files/ENCFF425LKJ/@@download/ENCFF425LKJ.bigWig',
+    },
+    {
+        file_format: 'bigWig',
+        output_type: 'plus strand signal of all reads',
+        accession: 'ENCFF638QHN',
+        href: '/files/ENCFF638QHN/@@download/ENCFF638QHN.bigWig',
+    },
+    {
+        file_format: 'bigWig',
+        output_type: 'plus strand signal of unique reads',
+        accession: 'ENCFF541XFO',
+        href: '/files/ENCFF541XFO/@@download/ENCFF541XFO.bigWig',
+    },
+    {
+        file_format: 'bigBed',
+        output_type: 'transcription start sites',
+        accession: 'ENCFF517WSY',
+        href: '/files/ENCFF517WSY/@@download/ENCFF517WSY.bigBed',
+    },
+    {
+        file_format: 'bigBed',
+        output_type: 'peaks',
+        accession: 'ENCFF026DAN',
+        href: '/files/ENCFF026DAN/@@download/ENCFF026DAN.bigBed',
+    },
+    {
+        file_format: 'bigBed',
+        output_type: 'peaks',
+        accession: 'ENCFF847CBY',
+        href: '/files/ENCFF847CBY/@@download/ENCFF847CBY.bigBed',
+    },
+];
+
+
+function rAssemblyToSources(assembly, region) {
```

Comment:
All these defaults should be moved to a local JSON configuration file.   Not sure in what directory it would go or what the best way of loading JSON data in but @forresttanaka will know.